### DISCUSSION TITLE
Connect to bluetooth device

### DISCUSSION THREAD

**Discussion by ghost** [asmvik/yabai#1820] [Ideas]
Is there a command to create a shortcut to instantly connect to a bluetooth device?

**Comment by PrayagS** [comment]
Not sure why you would discuss this here. `yabai` is a window manager.

What you're looking for is solved by something like `skhd` or `hammerspoon`.

**Comment by brorbw** [comment]
There is this https://github.com/lapfelix/BluetoothConnector. You would need the SSID of the device you would like to connect to. 

Then you can do:
```sh
ctrl + cmd - c: zsh -c "BluetoothConnector --connect FF-FF-FF-FF-FF-FF --notify"
```
